src/screens/ExchangeModal.js

```diff
@@ -133,22 +131,37 @@ const ExchangeModal = ({
   } = useUniswapAllowances();
   const { web3ListenerInit, web3ListenerStop } = useBlockPolling();
   const { uniswapAssetsInWallet } = useUniswapAssetsInWallet();
-  const { accountAddress, chainId, nativeCurrency } = useAccountSettings();
```

Comment:
To take into account while doing the rebase,   I'm using `accountAddress` for the max swap logic